### PR TITLE
feat: use embedded dashboard in docker release

### PR DESCRIPTION
**Pull request by BugenZhao**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Use embedded dashboard in docker release. Remove the arguments and related code for `dashboard-ui-path` as it's not necessary any more.

I find it really difficult to build the dashboard on `manylinux2014`, which is used to release the Linux binary, due to the extremely outdated glibc. Will investigate later, or we may choose to directly download the pre-built artifacts by GitHub CI.

Test: https://buildkite.com/risingwavelabs/docker/builds/15398

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Comment by kwannoel** [comment]
> I find it really difficult to build the dashboard on manylinux2014

So after this PR, linux release will temporarily not have dashboard?

**Review by kwannoel — APPROVED** [review]

File: src/meta/node/src/lib.rs [hunk 1]
```diff
@@ -83,9 +83,6 @@ pub struct MetaNodeOpts {
     #[clap(long, env = "RW_SQL_ENDPOINT")]
     pub sql_endpoint: Option<String>,
 
-    #[clap(long, env = "RW_DASHBOARD_UI_PATH")]
-    pub dashboard_ui_path: Option<String>,
```
Comment (xxchan) [inline src/meta/node/src/lib.rs:87]: Does cloud use such env var?

**Review by xxchan — APPROVED** [review]

**Comment by xxchan** [comment]
tried 
```
docker run -it --pull=always -p 4566:4566 -p 5691:5691 ghcr.io/risingwavelabs/risingwave:bz-embedded-dashboard
```

It works fine

File: src/meta/node/src/lib.rs (hunk 1 above)
Comment (BugenZhao) [reply src/meta/node/src/lib.rs:87]: I don't think so. We've already set it in Dockerfile. 😄

**Pull request merged**